pkg/arvo/app/talk.hoon

```diff
@@ -716,8 +716,12 @@
       ++  circ                                          ::  circle
         ;~  pose
           (cold incir col)
-          ;~(pfix cen (stag self urs:ab))
           ;~(pfix net (stag (^sein:title self) urs:ab))
+          ;~  pfix  cen
+            %+  stag  self
+            %+  sear  |=(circ=name ?~(circ ~ (some circ)))
```

Comment:
I might prefer an explicit `=('' circ)` here instead of a `?~` check, just to be clear about what kind of data we're checking here.

Don't think there's any reason to change the order of this wrt the other options in `++circ`, but it doesn't conflict with the other rule (pfix with `/`), so that's not really a problem.

Reply:
Fixed, and my logic for changing the order was to make the heavier form on the same indentation level come second, flowing downwards.